protocol/app/upgrades/v5.0.0/upgrade_container_test.go

```diff
@@ -1,362 +0,0 @@
-//go:build all || container_test
-
-package v_5_0_0_test
-
-import (
-	"testing"
-
-	consensus "github.com/cosmos/cosmos-sdk/x/consensus/types"
-	"github.com/cosmos/gogoproto/proto"
-	v_5_0_0 "github.com/dydxprotocol/v4-chain/protocol/app/upgrades/v5.0.0"
-	"github.com/dydxprotocol/v4-chain/protocol/dtypes"
-	"github.com/dydxprotocol/v4-chain/protocol/testing/containertest"
-	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
-	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
-	perpetuals "github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
-	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
-	vaulttypes "github.com/dydxprotocol/v4-chain/protocol/x/vault/types"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-const (
-	AliceBobBTCQuantums = 1_000_000
-	CarlDaveBTCQuantums = 2_000_000
-	CarlDaveETHQuantums = 4_000_000
-)
-
-func TestStateUpgrade(t *testing.T) {
```

Comment:
For my understanding, why do we need to delete this?

Reply:
It runs using the same container test setup I think, and will break since we're setting it up for a 5.0.5 to 5.1.0 upgrade and not a 4.0.0 to 5.0.0 upgrade.